Urban-Fun/View Controllers/HostViewController.m

```diff
@@ -72,6 +113,7 @@ - (IBAction)uploadActivity:(id)sender {
                                Activity *recentlyPosted = activity[0];
                                [[User currentUser] addObject:recentlyPosted forKey:@"activitiesHosted"];
                                [[User currentUser] saveInBackground];
+                               [self.hostVCDelegate didPostActivity:recentlyPosted];
```

Comment:
I'm not quite sure this is working properly, since you already dismissed the viewcontroller in line `105` and the delegate is a weak property maybe you're getting `nil` here and the callback is not being executed, did you try this code?

Reply:
deleted this line